modules/data.land/R/download.SM_CDS.R

```diff
@@ -0,0 +1,152 @@
+#' Download CDS soil moisture data for the SDA workflow.
+#'
+#' @param outfolder physical paths to where the unziped soil moisture files are downloaded.
+#' @param time.points A vector contains each time point within the start and end date.
+#' @param overwrite flag determine if we want to overwrite existing files when downloading.
+#' @param auto.create.key flag determine if we want to automatically create the credential file.
+#'
+#' @return A vector containing file paths to the downloaded files.
+#' @export
+#' 
+#' @examples
+#' @author Dongchen Zhang
+#' @importFrom dplyr %>%
+download.SM_CDS <- function(outfolder, time.points, overwrite = FALSE, auto.create.key = FALSE) {
+  ###################################Introduction on how to play with the CDS python API##########################################
```

Comment:
I'd recommend moving all of this instruction text out of the function and into the ROxygen as part of the Details section

Reply:
Fixed.